spec/system/search_game_rooms_spec.rb

```diff
@@ -2,18 +2,27 @@
 
 RSpec.describe 'SearchGameRooms', type: :system do
   include_context 'when user is logged in'
+  let(:user2) { create(:user, username: 'janedoe', email: 'janedoe@example.com') }
```

Comment:
Is it possible to add the second using Factory? I'm just wondering.

Reply:
@lea927 If the username is not significant, instead of creating another user, I think we can use the `sequence` method. So that, whenever we create a user; it's always distinct. It would be easy to create multiple users :)